code/modules/codex/codex_mob.dm

```diff
@@ -0,0 +1,20 @@
+
+/mob/proc/can_use_codex()
+	return TRUE
+
+/* 
+/mob/new_player/can_use_codex()
+	return TRUE
+
+/mob/living/silicon/can_use_codex()
+	return TRUE
+
+/mob/observer/can_use_codex()
+	return TRUE
+
+/mob/living/carbon/human/can_use_codex()
+	return TRUE //has_implant(/obj/item/implant/codex, functioning = TRUE)
+*/
+
+/mob/living/carbon/human/species/get_codex_value()
```

Comment:
```suggestion
/mob/living/carbon/human/get_codex_value()
```
`/mob/living/carbon/human/species` is a helper type used for mapping and adminspawn. Make it the normal human type, or better yet, move it to `carbon` since that's where DNA is defined.